fastlane/lib/fastlane/actions/get_version_number.rb

```diff
@@ -113,19 +113,19 @@ def self.available_options
         [
           FastlaneCore::ConfigItem.new(key: :xcodeproj,
                              env_name: "FL_VERSION_NUMBER_PROJECT",
-                             description: "optional, you must specify the path to your main Xcode project if it is not in the project root directory",
+                             description: "Path to the main Xcode project to read version number from, optional. By default will use the first Xcode project found within the project root directory",
```

Comment:
I actually don't like that this action selects the first project 😱 But I like the documentation that tells about this behavior. I think I'm going to create a follow up issue about fixing that to prompt for a project to select 🤔

Reply:
Created https://github.com/fastlane/fastlane/issues/14123 as a follow up to this

Reply:
Completely agree, thanks.